strider/trapi.py

```diff
@@ -283,3 +359,23 @@ def add_descendants(
     })
 
     return qg
+
+
+def build_unique_kg_edge_ids(message: Message):
+    """
+    Replace KG edge IDs with a string that represents
+    whether the edge can be merged with other edges
+    """
+    for edge_id in list(message["knowledge_graph"]["edges"].keys()):
```

Comment:
```suggestion
    # make a copy of the edge keys because we're about to change them
    for edge_id in list(message["knowledge_graph"]["edges"].keys()):
```